src/CONTRIBUTING.md

```diff
@@ -0,0 +1,154 @@
+* When using the "Zoey says" callouts, be sure to use the blockquote formatting, ensuring that an extra `>` is included between the "Zoey says..." text and the actual note content. This is required in order to properly convert the markdown into HTML.
+    * For example:
+        ```
+        > Zoey says...
+        >
+        > Here's a very helpful comment!
+        ```
+
+## Making changes to the tutorial code
+
+The workflow for updating the Super Rentals Tutorial involves working within two directories of this project: `/src/chapters` and `/dist`. The editable prose lives in `src/chapters`; any changes that are made to the prose should be done there. The tutorial uses the content in `/src/chapters` to generate the prose markdown that is output into `/dist/chapters`. Interspersed with the prose are commands that are used to generate the code for the _actual_ tutorial app, which is outputted at `/dist/code/super-rentals`.
+
+Once you have made your changes in the tutorial prose or code, you can run a few different commands to verify that your changes render as expected.
+
+* To regenerate all of the chapters (and rebuild the app), run `yarn build`. This will `rm -rf` anything previously generated in the `/dist` directory.
+* To generate a single chapter without removing all the previously generated content, run `yarn generate src/chapters/00-your-chapter-to-edit-here.md`. This will rebuild _just_ the chapter that is specified without deleting everything else in the `/dist` directory.
+
+### Code changes
+
+To make code changes, you will first need to locate the chapter you wish to edit in `/src/chapters`. Depending on the type of code change you'd like to make, you will need to use different commands.
+
+#### Running commands
+
+If you'd like to run a command, you will need to use the `run: command`. Be sure to include `cwd=super-rentals` so that the file is generated in the app's directory.
+
+For example:
+
+    ```run:command hidden=true cwd=super-rentals
+    ember generate component rental
+    ```
+
+#### Creating files
+
+To create a file, you will need to use the `run:file:create` command. Be sure to specify the language, the filename, as well as the `cwd` directory.
+
+For example:
+
+    ```run:file:create lang=handlebars cwd=super-rentals filename=app/templates/about.hbs
+    <h2>About Super Rentals</h2>
+    ```
+
+After creating a file, you should be sure to _add_ it shortly afterwards so that it is tracked by git, and will be committed in the process of generating the tutorial app. This can be a `hidden` command, since it is not part of the prose of the tutorial.
+
+For example:
+
+    ```run:command hidden=true cwd=super-rentals
+    git add app/templates/about.hbs
+    ```
+
+If you are adding a _test_ file, be sure to run `yarn test` before adding the test file. This way, the generator will run the tests before adding them, and will subsequently fail in generating broken code if the tests themselves fail.
```

Comment:
Remove “subsequently” maybe. It sounds like the failure is deferred till the end of the run.